stlab/concurrency/default_executor.hpp

```diff
@@ -19,20 +19,16 @@
 
 #if STLAB_TASK_SYSTEM(LIBDISPATCH)
 #include <dispatch/dispatch.h>
-#elif STLAB_TASK_SYSTEM(EMSCRIPTEN)
```

Comment:
Did you consider moving the removal of the EMSCRIPTEN task system into its own PR?

Reply:
Done in #479.